### PR TITLE
Add CVE-2015-5243: phpWhois PHP Code Injection

### PR DESCRIPTION
**Pull request by lxp**
Security advisory: https://github.com/sbaresearch/advisories/tree/public/2018/SBA-ADV-20180425-01_phpWhois_Code_Execution

### CODE REVIEW

File: brightlocal/phpwhois/CVE-2015-5243.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title:     PHP Code Injection
+link:      https://github.com/sbaresearch/advisories/tree/public/2018/SBA-ADV-20180425-01_phpWhois_Code_Execution
+cve:       CVE-2015-5243
+branches:
+    master:
+        time:       null
+        versions:   ['<=4.2.5']
+reference: composer://brightlocal/phpwhois
```
Comment (Ocramius) [inline brightlocal/phpwhois/CVE-2015-5243.yaml]: Why are forks included in the advisory?

Comment (stof) [reply brightlocal/phpwhois/CVE-2015-5243.yaml]: well, if these projects are registered on packagist under their own names, it totally makes sense to mark them as vulnerable.

Comment (lxp) [reply brightlocal/phpwhois/CVE-2015-5243.yaml]: They are all registered on packagist. I determined vulnerable projects by searching for phpwhois on the packagist.org and then manually inspecting each project, if they contain the vulnerable lines of code.

**Comment by stof** [comment]
Please fix the advisories to make them valid.

**Comment by Ocramius** [comment]
Would probably use `*`

**Comment by lxp** [comment]
I tried to satisfy the validator with expressions like `*`, `<*`, `<=*` but nothing worked.
The regex of the validator just does not allow that. Therefore, I removed the three projects without releases.
If you decide on a way to report projects without releases, please let me know.

**Comment by lxp** [comment]
Is there still an issue blocking the merge of this pull request?

**Pull request merged**